cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -3087,6 +3087,21 @@ pDRepActivity =
     , Opt.help "DRep activity period, in epochs."
     ]
 
+toNonNegativeIntOrErr :: Rational -> L.NonNegativeInterval
```

Comment:
There is `toNonNegativeIntervalOrErr` in  `Cardano.CLI.EraBased.Options.Governance.Actions` already. I think we should move it here.